e2e_test.ts

```diff
@@ -34,55 +34,119 @@ import options from "./fresh.config.ts";
  */
 const handler = await createHandler(manifest, options);
 
-function assertResponseNotFound(resp: Response) {
+function assertResponseAccepted(resp: Response) {
+  assert(resp.ok);
+  assertEquals(resp.body, null);
+  assertEquals(resp.status, Status.Accepted);
+}
+
+function assertResponseBadRequest(resp: Response) {
   assertFalse(resp.ok);
-  assertEquals(resp.status, Status.NotFound);
+  assertEquals(
+    resp.headers.get("content-type"),
+    "text/plain;charset=UTF-8",
+  );
+  assertEquals(resp.status, Status.BadRequest);
+}
+
+function assertResponseCreated(resp: Response) {
+  assert(resp.ok);
+  assertEquals(resp.body, null);
+  assertEquals(resp.status, Status.Created);
+}
+
+function assertResponseHtml(resp: Response) {
+  assert(resp.ok);
+  assertInstanceOf(resp.body, ReadableStream);
+  assertEquals(
+    resp.headers.get("content-type"),
+    "text/html; charset=utf-8",
+  );
+  assertEquals(resp.status, Status.OK);
 }
 
 function assertResponseJson(resp: Response) {
   assert(resp.ok);
   assertNotEquals(resp.body, null);
   assertEquals(resp.headers.get("content-type"), "application/json");
+  assertEquals(resp.status, Status.OK);
 }
 
-Deno.test("[e2e] GET /", async () => {
-  const resp = await handler(new Request("http://localhost"));
+function assertResponseNoContent(resp: Response) {
+  assert(resp.ok);
+  assertEquals(resp.body, null);
+  assertEquals(resp.status, Status.NoContent);
+}
 
+function assertResponseNotFound(resp: Response, header: string) {
+  assertFalse(resp.ok);
+  assertEquals(
+    resp.headers.get("content-type"),
+    header,
+  );
+  assertEquals(resp.status, Status.NotFound);
+}
+
+function assertResponseRedirect(
+  resp: Response,
+  location: string,
+  status: Status,
+) {
+  assertFalse(resp.ok);
+  assertFalse(resp.body);
+  assertStringIncludes(resp.headers.get("location")!, location);
+  assertEquals(resp.status, status);
+}
```

Comment:
```suggestion
function assertRedirect(response: Response, location?: string) {
  assert(isRedirectStatus(response.status));
  if (location !== undefined) {
    assertEquals(response.headers.get("location"), location);
  } else {
    assert(response.headers.has("location"));
  }
}
```
We can use [`isRedirectStatus()`](https://deno.land/std@0.202.0/http/mod.ts?s=isRedirectStatus).

Reply:
`std` is such a beast.